onvm/go.sh

```diff
@@ -48,13 +48,32 @@ fi
 
 shift 3
 
+# Verify that bc is installed
+if [[ $(command -v bc | grep -c "/usr/bin/bc") == 0 ]]
```

Comment:
Back to Tim's point about checking it's installed. `bc` doesn't need to be installed at /usr/bin/bc to work, it just needs to be installed. If we do something like `which`, if there is no output, `bc` isn't installed. Kind of off topic, for any of you that are interested in bash, this is an interesting take on the speed of `-z` in bash scripts.
https://stackoverflow.com/a/35165216/10158041

Reply:
Good idea. That improvement is in the newest commit for this PR and #219.